--- ui/hud_complex_plane.gdshader
+++ ui/hud_complex_plane.gdshader
@@ -71,8 +71,8 @@
     vec3 color = base * (albedo + emission ) * 2.0;

     // Draw unit circle
     float unit_circle = abs(mag - 1.0);
-    float circle_mask = smoothstep(0.02 * scale + fwidth(unit_circle), 0.0, unit_circle);
+    float circle_mask = smoothstep(0.03 * scale, 0.0, unit_circle);
     color = mix(color, vec3(0.9), circle_mask);

     // Draw arrow for phase
